CondFormats/CTPPSReadoutObjects/xml/mapping_timing_diamond_2017.xml

```diff
@@ -0,0 +1,129 @@
+<top>   
```

Comment:
hello @forthommel 

are you planning on moving this mapping content to the conditions database ?

I guess you need CTPPSPixelDAQMappingRcd 
to be provided to you from this parallel pr https://github.com/cms-sw/cmssw/pull/18359/files#diff-03e8ca1cea42e0df6e5deb53fc5dd4be ?